### PR TITLE
Unblock the repo using our forked version

### PR DESCRIPTION
**Pull request by sdake**
Our forked version of gogo-protobuf needs specific tagging and a process
of merging when updating the gogo-protobuf version.  More documetation
to follow on the developer wiki.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by sdake** [comment]
https://github.com/istio/istio/pull/15928

**Comment by sdake** [comment]
https://github.com/istio/tools/pull/233

**Comment by sdake** [comment]
https://github.com/istio/api/pull/1007

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ replace k8s.io/klog => github.com/istio/klog v0.0.0-20190424230111-fb7481ea8bcf
 
 replace github.com/spf13/viper => github.com/istio/viper v1.3.3-0.20190515210538-2789fed3109c
 
-replace github.com/gogo/protobuf => github.com/istio/gogo-protobuf v1.2.2-0.20190726125433-4c9abdb3090c
+replace github.com/gogo/protobuf => github.com/istio/gogo-protobuf v1.2.2
```
Comment (sdake) [inline go.mod]: Does this not force the usage of v1.2.2 of our tag?

Comment (rshriram) [reply go.mod]: I think yes.

**Comment by rshriram** [comment]
Now that unit tests have passed, can we force merge ?

**Comment by geeknoid** [comment]
Let's wait till everything passes please...

**Pull request merged**